packages/block-library/src/social-link/edit.js

```diff
@@ -6,18 +6,23 @@ import classNames from 'classnames';
 /**
```

Comment:
I see a few strings here where we use Title Case. I think we've been recently moving to always using "Sentence case" in all of our strings. @karmatosed do you confirm?

Reply:
<img width="283" alt="Screen Shot 2019-12-26 at 10 03 00 AM" src="https://user-images.githubusercontent.com/9565066/71483846-f378a780-27c6-11ea-83f6-de34fa5dc0ca.png">

@youknowriad that doesn't appear to be the standard used for Panel Titles and that doesn't seem to be the standard [outlined in the docs](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/copy-guide.md#five-pay-attention-to-capitalization). Was there a discussion about that I missed? If that's changed, the docs need to be updated and issues opened to address the many instances of title case throughout the application.

Reply:
I've since dug and found the PRs and issue discussion around sentence case. A major departure like that should be memorialized in docs well before it's put into practice, to avoid confusing situations like this.

Reply:
I don't know to be honest, I'll let @karmatosed chime in when she comes back. I think we can land this as is for now.